charts/dbildungs-iam-ldap/templates/_helpers.tpl

```diff
@@ -217,7 +149,7 @@ Return the proper base domain
 tmp method to iterate through the ldapDomain
 */}}
 {{- define "tmp.baseDomain" -}}
-{{- if regexMatch ".*=.*" .Values.global.ldapDomain }}
+{{- if regexMatch ".*=.*,.*" .Values.global.ldapDomain }}
```

Comment:
https://github.com/jp-gouin/helm-openldap/commit/78f801c07f6f6477e48fd3716bdbe105607d2ee9 this has been introduced in the Helm Chart